modules/understanding-upgrade-channels.adoc

```diff
@@ -33,15 +60,25 @@ for more build information.
 [id="fast-version-channel_{context}"]
 == fast-{product-version} channel
 
-The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
+The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability (GA) release. The `fast-{product-version}` channel contains releases as soon as their errata are published.
+
+As such, these releases are fully supported and are production quality. The releases in this channel are closely monitored using a Telemetry system. The releases in the `fast-{product-version}` channel are promoted from `candidate-{product-version}` after the builds have performed well. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel.
+
+[NOTE]
+====
+Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
+====
 
 You can use the `fast-{product-version}` channel to upgrade from a previous minor version of {product-title}.
 
+Example channel name: `fast-{product-version}`.
+
 ifndef::openshift-origin[]
 
 [id="stable-version-channel_{context}"]
 == stable-{product-version} channel
-While the `fast-{product-version}` channel contains releases as soon as their errata are published, releases are added to the `stable-{product-version}` channel after a delay. During this delay, data is collected from Red Hat SRE teams, Red Hat support services, and pre-production and production environments that participate in connected customer program about the stability of the release.
+
+The `stable{product-version}` channel contains general availability releases that have a proven track record of upgrade success. The builds were originally a part of the `fast-{product-version}` channel where data was collected by Red Hat to ensure the stability of the release. These builds are supported by Red Hat.
```

Comment:
I think this new paragraph makes "we cooked in fast for a while" point less clear by dropping the word "delay".  We are not committing to a specific amount of delay, but we certainly field lots of internal questions like "when will 4.y.z be in stable-4.y?" and the response is basically "it will be in stable channels when we feel like it's stable.  In the meantime, fast-4.y is also monitored and fully supported, and you can do your own testing, so if you feel like you're ready and we're dragging our feet, feel free to switch to fast-4.y".  So fast-4.y gets you low-latency update recommendations, as soon as Red Hat is ready to start supporting those updates.  And stable-4.y gives you high-latency update recommendations, where we are not only ready to support them, but are also fairly confident that bugs that take a week or two to bubble out, or bugs that only affect some rarer cluster configurations, and whatnot have also had time to get noticed and triaged.